packages/ra-ui-materialui/src/button/SaveButton.spec.tsx

```diff
@@ -44,7 +53,9 @@ describe('<SaveButton />', () => {
             <TestContext>
                 <ThemeProvider theme={theme}>
                     <SaveContextProvider value={saveContextValue}>
-                        <SaveButton {...invalidButtonDomProps} />
+                        <FormContextProvider value={formContextValue}>
```

Comment:
I don't understand why this is now required and it worries me (that it's now required). this will be a gotcha for users.

Reply:
I actually don't get how those tests could have pass before